docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,154 @@
+
+Create the following two packages:
+
+- YOUR_VEHICLE_launch (see [here](https://github.com/autowarefoundation/sample_vehicle_launch) for example)
+- YOUR_SENSOR_KIT_launch (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch) for example)
+
+It is recommended you write the above two packages in `autoware.repos` file of your meta-repository.
+
+### Adapt YOUR_VEHICLE_launch for autoware launching system
+
+#### At YOUR_VEHICLE_description
+
+Define URDF and parameters in the package (see [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description) for example).
+
+#### At YOUR_VEHICLE_launch
+
+Create a launch file (see [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_launch) for example).
+If you have multiple vehicles with similar hardware setup, you can specify `vehicle_id` to distinguish them.
+
+### Adapt YOUR_SENSOR_KIT_description for autoware launching system
+
+#### At YOUR_SENSOR_KIT_description
+
+Define URDF and extrinsic parameters for all the sensors here (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description) for example).
+Note that you need to calibrate extrinsic parameters for all the sensors beforehand.
+
+#### At YOUR_SENSOR_KIT_launch
+
+Create `launch/sensing.launch.xml` that launches the interfaces of all the sensors on the vehicle. (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_launch) for example).
+
+!!! note
+
+    At this point, you are now able to run planning_simulator.
+    If you want to try, you may install Autoware (follow [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/)) and run the following command:
+
+    ```bash
+    ros2 launch autoware_launch planning_simulator.launch.xml vehicle_model:=YOUR_VEHICLE sensor_kit:=YOUR_SENSOR_KIT map_path:=/PATH/TO/YOUR/MAP
+    ```
+
+## 5. Create a `vehicle_interface` package
+
+You need to create an interface package for your vehicle.
+The package is expected to provide the following two functions.
+
+1. Receive command messages from `vehicle_cmd_gate` and drive the vehicle accordingly
+2. Send vehicle status information of the vehicle to Autoware
+
+You can find detailed information about the requirements of `vehicle_interface` [here](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/vehicle-interface/).
+You can also refer to [pacmod_interface](https://github.com/tier4/pacmod_interface) as an example.
+
+## 6. Launch Autoware
+
+This section briefly explains how to run your vehicle with Autoware.
+
+### Install Autoware
+
+Follow the step [here](https://autowarefoundation.github.io/autoware-documentation/pr-86/installation/autoware/).
```

Comment:
* Changed URL to point to the main documentation branch rather than pr-86.
* Added a note that users should clone their forked Autoware repository and not the one specified in the documentation

```suggestion
Follow the steps [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/), remembering to clone your forked Autoware repository!
```